libraries/nginx_service.rb

```diff
@@ -94,6 +96,7 @@ def action_enable
       directory '/run/nginx' do
```

Comment:
Yea, that's possibly it. I will try changing the permissions on the erroneous directories in the recipe and see if that works out